config/vufind/geofeatures.ini

```diff
@@ -1,8 +1,15 @@
 ; Geographic feature configuration settings
 ;
-; Currently all but the basemap configurations for the geographic feature
-; functionality are located in config.ini (for Map Tab display) and in
-; searches.ini (for Map Selection).
+; Legacy geographic feature configurations are located in
+; config.ini (for Map Tab display) and in searches.ini (for Map Selection).
+; Configuration settings in this file overwrite settings in the
+; config.ini and searches.ini files.
+
+[GeoPlatform]
+; Configures the mapping platform that should be used.
+; Replaces config.ini recordMap setting.
+; Options:  leaflet (default), openlayers (legacy)
```

Comment:
How long do you plan on maintaining openlayers as a legacy platform? Is it worth doing?

Reply:
I'd prefer not to maintain it if possible, and honestly, I'm don't think it's really worth it given the improvements with the leaflet code.  I'm happy to make this a clean transition and remove all the OpenLayers code with this PR if you think it's advisable.

Reply:
If that's the end goal, I think it's probably simpler to just go ahead and do it now.

Reply:
Ok. Will do.